frontend/src/utils/admin-utils.ts

```diff
@@ -63,6 +63,7 @@ export const getChartAdminBoundaryParams = (
         code: properties[item.id],
         level: item.level,
         name: properties[item.name],
+        localName: properties[`${item.name}_`] || properties[item.name],
```

Comment:
what's with the `${item.name}_` ?

Reply:
I have updated `levels` so we get the correct property each time